src/encoded/static/components/footer.js

```diff
@@ -2,52 +2,50 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 
-// Reworking the data triggers to use buttons doesn't seem worth it to avoid an eslint warning.
-/* eslint-disable jsx-a11y/anchor-is-valid, react/prefer-stateless-function */
-export default class Footer extends React.Component {
-    render() {
-        const session = this.context.session;
-        const disabled = !session;
-        let userActionRender;
-
-        if (!(session && session['auth.userid'])) {
-            userActionRender = <a href="#" data-trigger="login" disabled={disabled}>Submitter sign-in</a>;
-        } else {
-            userActionRender = <a href="#" data-trigger="logout">Submitter sign out</a>;
-        }
-        return (
-            <footer id="page-footer">
+/* eslint-disable jsx-a11y/anchor-is-valid */
```

Comment:
It is great you made this a stateless function as state is not needed; so there may be a performance boost in future versions of React